lib/AST/Decl.cpp

```diff
@@ -4023,7 +4023,8 @@ static AccessScope
 getAccessScopeForFormalAccess(const ValueDecl *VD,
                               AccessLevel formalAccess,
                               const DeclContext *useDC,
-                              bool treatUsableFromInlineAsPublic) {
+                              bool treatUsableFromInlineAsPublic,
+                              bool applyImportAccessLevel = false) {
```

Comment:
Is it more common to want to pass `applyImportAccessLevel = false`? My inclination would be to default this to `true` to ensure applying the restriction is not accidentally forgotten.

Reply:
You are very right, thanks for brining it up! The function ended up not even using this parameter. It actually goes with what I said above, we want to apply the import restriction as long as there's a `useDC` so we don't need a separate parameter for it.

Reply:
I simply removed the parameter.